litestar/types/helper_types.py

```diff
@@ -15,9 +17,12 @@
     Union,
 )
 
+from litestar.response.sse import ServerSentEventMessage
```

Comment:
CI fails for a weird reason to me, mentionning circular imports (https://github.com/litestar-org/litestar/actions/runs/7713504937/job/21023551864?pr=3048) and this is the only instance  can think in the PR could cause this, however at first  put him under TPYE_CHECKING but ruff was complaining

Reply:
if I put it under TPYE_CHECKING I get:
litestar/types/helper_types.py:23:39: TCH004 Move import `litestar.response.sse.ServerSentEventMessage` out of type-checking block. Import is used for more than type hinting.

Reply:
> Import is used for more than type hinting.

You'll need to put the import in quotes. I left a comment down below!